README.md

```diff
@@ -95,6 +95,9 @@ Once enough people have reached the point that they feel comfortable with basic
 
 * [Linux Terminal](https://ubuntu.com/tutorials/command-line-for-beginners)
 * [Getting Started with Git and GitHub](https://product.hubspot.com/blog/git-and-github-tutorial-for-beginners)
+  * Add to the start of step 6 where one uses "git checkout master" then adds "git branch -M main" before proceeding
+  * Change something before pushing in step 7
+  * Change in step 10 to "git pull origin main" accordingly
```

Comment:
don't know what you mean by change in step 10, the goal here isn't to make a todo list for the author of the tutorial but rather instructions for the users of the tutorial.